openlibrary/templates/account/books.html

```diff
@@ -65,6 +67,17 @@ <h1 style="display:inline">
 
       $if owners_page:
         <a href="$ctx.path/stats" title="$('View stats about this shelf')">$_('Stats')</a>
+      
+      <span class="mytools-tools"><img src="/images/icons/icon_sort.png" alt="$_('Sorting by')" style="margin-right:10px;" width="9" height="11">
```

Comment:
Oops, my typo; my bad :P
```suggestion
      <span class="mybooks-tools"><img src="/images/icons/icon_sort.png" alt="$_('Sorting by')" style="margin-right:10px;" width="9" height="11">
```

Reply:
Ah we'll also want to move this a little lower; just before the `macros.Pages` section:

![image](https://user-images.githubusercontent.com/6251786/123575063-f75ffe00-d79e-11eb-9530-47cb89d183e7.png)